actuators:
 hip:
  axial_MoI: 7.6e-4 # approximate axial moment of inertia of the rotor ([Kg m^2]) 
  mass: 0.27 # mass of the actuator ([Kg])
  
  red_ratio: 0.02 # gear reduction ratio of the actuator (always <=1)
  
  efficiency: 0.8 # harmonic gear efficiency (0-1)
  
  K_t: 0.045 # torque constant ([Nm/A])
  K_bmf: 4.71 # ([V/Krpm])
  
  R: 0.09  # resistance (per phase?) ([Ohm])
  
  tau_max_br: 1.1 # maximum continuous torque before reduction ([Nm])
  tau_max_ar: 44 # maximum continuous torque after reduction ([Nm])
  
  tau_peak_br: 2.9 # allowed peak torque before reduction ([Nm])
  tau_peak_ar: 112 # allowed peak torque after reduction ([Nm])
  
  I_max: 24 # current @ tau_max_br ([A])
  I_peak: 62 # current @ tau_peak ([A])
  
  omega_max_nl_bf: 1067.2 # maximum no-load velocity @ 48 V, before reduction ([rad/s])
  omega_max_nl_af: 21.3 # maximum no-load velocity @ 48 V, after reduction ([rad/s])
  omega_max_nl_af44: 20.3 # maximum no-load velocity @ 48 V, @ 44 Nm, after reduction ([rad/s])
  omega_max_nl_af112: 18.8 # maximum no-load velocity @ 48 V, @ 112 Nm, after reduction ([rad/s])
  
  
 knee:
  axial_MoI: 7.6e-4 # approximate axial moment of inertia of the rotor ([Kg m^2]) 
  mass: 0.27 # mass of the actuator ([Kg])
  
  red_ratio: 0.02 # gear reduction ratio of the actuator (always <=1)
  
  efficiency: 0.8 # harmonic gear efficiency (0-1)
  
  K_t: 0.045 # torque constant ([Nm/A])
  K_bmf: 4.71 # ([V/Krpm])
  
  R: 0.09  # resistance (per phase?) ([Ohm])
  
  tau_max_br: 1.1 # maximum continuous torque before reduction ([Nm])
  tau_max_ar: 44 # maximum continuous torque after reduction ([Nm])
  
  tau_peak_br: 2.9 # allowed peak torque before reduction ([Nm])
  tau_peak_ar: 112 # allowed peak torque after reduction ([Nm])
  
  I_max: 24 # current @ tau_max_br ([A])
  I_peak: 62 # current @ tau_peak ([A])
  
  omega_max_nl_bf: 1067.2 # maximum no-load velocity @ 48 V, before reduction ([rad/s])
  omega_max_nl_af: 21.3 # maximum no-load velocity @ 48 V, after reduction ([rad/s])
  omega_max_nl_af44: 20.3 # maximum no-load velocity @ 48 V, @ 44 Nm, after reduction ([rad/s])
  omega_max_nl_af112: 18.8 # maximum no-load velocity @ 48 V, @ 112 Nm, after reduction ([rad/s])
